groups:
  turret:
    operating_mode: position
    profile_type: time
    profile_velocity: 1000
    profile_acceleration: 500               
    torque_enable: true
